jgiven-core/src/main/java/com/tngtech/jgiven/report/ReportGenerator.java

```diff
@@ -9,6 +9,9 @@
 import com.tngtech.jgiven.report.asciidoc.AsciiDocReportGenerator;
 import com.tngtech.jgiven.report.text.PlainTextReportGenerator;
 
+import javax.annotation.Nullable;
```

Comment:
Do you know from which dependency this Nullable annotation comes from? I would leave that out if it introduces some new dependency.

Reply:
javax should be pretty much standard library. Unfortunately I found this article: https://jaxenter.de/jakarta-ee-java-ee-javax-namespace-83430 which implies that the namespace is actually deprecated :(
I guess I'll just remove it. Its not super useful in the first place